src/components/firstProfile/name/NameInput.tsx

```diff
@@ -53,18 +45,28 @@ const NameInput = () => {
         className={classes.profileInput}
         value={fullName}
         onChange={(e) => handleInputChange(e.target.value)}
-        error={!!error}
         fullWidth
       />
-      {!hasTyped && (
-        <FormHelperText>
-          {`Please, note - you won’t be able to change this field later`}
+
+      {/* Error message for invalid name */}
+      {hasError && (
+        <FormHelperText className={classes.errorBox}>
+          <h4>Your name</h4>
```

Comment:
There is a Warning in this place
![image](https://github.com/user-attachments/assets/3bc51e71-2100-4d43-8a52-8785ccd8ae67)

It's about this structure
![image](https://github.com/user-attachments/assets/715db29c-a73d-457a-8a66-b6b53bc62282)

It can be solved by `<FormHelperText className={classes.errorBox} component="div">`

Reply:
Thank you! Done.